ed25519/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "ed25519"
+version = "0.0.1"
+authors = ["Jared Wasinger <j-wasinger@hotmail.com>"]
+license = "Apache-2.0"
+repository = "https://github.com/ewasm/ewasm-precompiles"
+description = "Ethereum ed25519 precompile in Rust"
+publish = false
+
+[dependencies]
+ewasm_api = { git = "https://github.com/ewasm/ewasm-rust-api", tag = "0.4.0" }
+parity-bytes = { git = "https://github.com/paritytech/parity-common" }
+ed25519_verify = { path = "ed25519_verify" }
```

Comment:
This doesn't actually needs to be a crate. It can be a separate file in the same directory.

Reply:
How do you run tests in that case?  I've tried that approach and cargo always complains about "main" being defined multiple times.

Reply:
That makes it so I can't run tests:
```
error: entry symbol `main` defined multiple times
  |
  = help: did you use #[no_mangle] on `fn main`? Use #[start] instead
```

Reply:
I'm not sure what you are doing. Just name it `verify.rs` and add `mod verify` in `lib.rs`.

Reply:
Sorry, I get the problem now. This is annoying.

Reply:
Workaround it with disabling `main` if running tests:
`[if(cfg(not(test))]`